run.js

```diff
@@ -4,7 +4,7 @@ const { pass, fail } = require('create-jest-runner');
 module.exports = ({ testPath }) =>
   new Promise((resolve) => {
     const start = new Date();
-    exec(`flow check-contents < ${testPath}`, (err, stdout) => {
+    exec(`flow ${testPath}`, (err, stdout) => {
```

Comment:
I think this will run flow on the imports as well. Will we get errors from dependencies as well? (Let's say we have c.js dependent on b.js dependent on a.js. Will a.js errors displayed on the c.js entry inside the jest runner?)